service/k8sclient/k8sclient.go

```diff
@@ -526,6 +526,12 @@ func (c *K8sClient) CreatePXCCluster(ctx context.Context, params *PXCParams) err
 		podSpec.VolumeSpec = c.volumeSpec(params.ProxySQL.DiskSize)
 	} else {
 		res.Spec.HAProxy = new(pxc.PodSpec)
+		if params.HAProxy.ComputeResources == nil {
+			params.HAProxy.ComputeResources = &ComputeResources{
+				MemoryBytes: "500M",
+				CPUM:        "500m",
+			}
+		}
```

Comment:
I think it's better to keep this logic on the pmm-managed side where we set default values.

Reply:
I agree. Closing it then